style.css

```diff
@@ -241,6 +242,24 @@ main{
     align-items: center;
     padding: 0 1rem;
     margin: 0.5rem 0;
+    border-radius: 5px;
+}
+
+.outlined-red{
+    background-color: white;
+    border: 1px solid #e74c3c;
+}
+.outlined-yellow{
+    background-color: white;
+    border: 1px solid #d2ab21;
+}
+.outlined-blue{
+    background-color: white;
+    border: 1px solid #3498db;
+}
+.outlined-green{
+    background-color: white;
+    border: 1px solid #2ecc71;
```

Comment:
Suggest to use variable names for colors to make the future changes easier.

Reply:
Sure will do that, thanks for recommending.